yt_dlp/extractor/tiktok.py

```diff
@@ -793,102 +792,150 @@ def _real_extract(self, url):
 
 class TikTokUserIE(TikTokBaseIE):
     IE_NAME = 'tiktok:user'
-    _VALID_URL = r'https?://(?:www\.)?tiktok\.com/@(?P<id>[\w\.-]+)/?(?:$|[#?])'
-    _WORKING = False
+    _VALID_URL = [
+        r'https?://(?:www\.)?tiktok\.com/@(?P<id>[\w\.-]+)/?(?:$|[#?])',
+        r'tiktokuser:(?P<id>MS4wLjABAAAA[\w-]{64})',
```

Comment:
We could do something like this if you think it's useful
```suggestion
        r'tiktokuser:(?P<id>MS4wLjABAAAA[\w-]{64})(?:@(?P<username>[\w.-]))?',
```